doc/nrf/releases/release-notes-1.9.0.rst

```diff
@@ -180,7 +180,7 @@ nRF9160: Asset Tracker v2
     * :ref:`asset_tracker_v2_cloud_module`
     * :ref:`api_cloud_wrapper`
     * :ref:`asset_tracker_v2_data_module`
-    * :ref:`asset_tracker_v2_gnss_module`
+    * GNSS module
```

Comment:
```suggestion
    * GNSS module (removed in 2.1.0)
```

Reply:
Shouldn't this be `removed in 2.2.0`?

Reply:
@coderbyheart Please check this and resolve if the response is fine or comment why it's not.

Reply:
Yes. Updated.